static/js/vue-cdr-access/src/components/filterTags.vue

```diff
@@ -24,7 +24,7 @@ const TYPES = {
     created: 'Date Created',
     contributorIndex: 'Contributor',
     format: 'Format',
-    subjectIndex: 'Subject',
```

Comment:
I think we want to also allow subjectIndex as a tag (it could also display as "Subject"), since users will be able to facet or search subjects.